### PR TITLE
Use secret.path rather than secret.id in data link editor

### PR DESCRIPTION
**Pull request by MrFlashAccount**
#### Tl;dr 
Closes: enso-org/cloud-v2#1177

This PR fixes a bug when we send secret.id instead of secret.path while creating/editing a Data link

#### Test Plan:

1) try to create a new Data link, choose type "S3", in next dropdown choose "AWS access key", then in access key id choose "enso secret". Dropdown should suggest paths.
2) try to edit a Data link, choose type "S3", in next dropdown choose "AWS access key", then in access key id choose "enso secret". Dropdown should suggest paths.

---

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [ ] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Review by indiv0 — APPROVED** [review]
Code changes LGTM

**Comment by indiv0** [comment]
✅ QA passed

Paths appear in dropdown correctly :)

image
